templates/partials/committee-summary.html

```diff
@@ -26,7 +26,7 @@ <h4 class="figure__title">Totals for {{totals.cycle|fmt_year_range|default("unav
       <div class="table__cell">{{reports.debts_owed_by_committee|currency|default("unavailable")}}</div>
     </div>
     <div class="table__row--note">
-      Source: {{reports.report_type_full|fmt_report_desc|default("unavailable")}}
+      Source: <a href="http://docquery.fec.gov/pdf/{{ committee.reports[0].beginning_image_number|last_n_characters }}/{{ committee.reports[0].beginning_image_number }}/{{ committee.reports[0].beginning_image_number }}.pdf">{{reports.report_type_full|fmt_report_desc|default("unavailable")}} {{ reports.report_year }}</a>
```

Comment:
`reports` here is an alias for `committee.reports[0]` (see the `set` line at the top), so there's no need to use the latter.